packages/dataviews/src/filter-summary.js

```diff
@@ -167,19 +190,82 @@ export default function FilterSummary( {
 				toggleRef.current?.focus();
 			} }
 			renderToggle={ ( { isOpen, onToggle } ) => (
-				<Button
-					__experimentalIsFocusable
-					size="compact"
-					onClick={ onToggle }
-					aria-expanded={ isOpen }
-					ref={ toggleRef }
-				>
-					<FilterText
-						activeElement={ activeElement }
-						filterInView={ filterInView }
-						filter={ filter }
-					/>
-				</Button>
+				<div className="dataviews-filter-summary__chip-container">
+					<Tooltip
+						text={ sprintf(
+							/* translators: 1: Filter name. */
+							__( 'Filter by: %1$s' ),
+							filter.name.toLowerCase()
+						) }
+						placement="top"
+					>
+						<div
+							className={ classnames(
+								'dataviews-filter-summary__chip',
+								{
+									'has-reset': canResetOrRemove,
+									'has-values': hasValues,
+									'is-primary': isPrimary,
+								}
+							) }
+							role="button"
+							tabIndex={ 0 }
+							// eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role
```

Comment:
Is this rule necessary? I see the div has a role, tabIndex, and both click/keyboard events. I'd think this rule was not triggered 🤔

Reply:
TIL.  Thank you!